### PR TITLE
[4.3.0] Add a reason for adding the role

### PR DESCRIPTION
**Pull request by shnrndk**
Fixes https://github.com/wso2/docs-apim/issues/7814

### CODE REVIEW

File: en/docs/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ In a multi-tenanted API Manager setup, self-signup to the Developer Portal works
 
 5. Sign in to the admin portal  ( `https://<HostName>:9443/admin` ) as tenant admin and navigate to **Advance Configurations** by clicking **Advance** under **Settings**.
 
-6. Add the following configuration and save the content.
+6. Add the following configuration and save the content. This was added because even though the new user can create a new account, that new user can't login to devportal without the `Internal/subscriber` role.
```
Comment (tharikaGitHub) [inline en/docs/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md:48]: Let's reword this as follows.

The `SignUpRoles` configuration indicates the default roles that will be assigned to a user who self signs up to the Developer portal.

Comment (shnrndk) [reply en/docs/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md]: Fixed in https://github.com/wso2/docs-apim/pull/7841/commits/2183d77603e476c853753b91b9f68caa7f69692e

**Review by ashera96 — APPROVED** [review]
